src/components/Tooltip/Tooltip.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {toolTipDriverFactory} from './Tooltip.driver';
+import {Tooltip} from './';
+import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
+
+describe('Tooltip', () => {
+  const createDriver = createDriverFactory(toolTipDriverFactory);
+
+  it('renders BO tooltip', () => {
+    const tooltip = createDriver(<Tooltip />);
+    expect(tooltip.exists()).toBeTruthy();
+  });
+
+  it('remains open when hovered out if relevant property is provided', () => {
```

Comment:
Add test for when shouldCloseOnClickOutside is false ?

Reply:
this functionality is tested in the core